src/main/kotlin/no/nav/syfo/kartleggingssporsmal/infrastructure/kafka/kartleggingssporsmalsvar/KartleggingssporsmalSvarConsumerTask.kt

```diff
@@ -20,9 +20,9 @@ fun launchKartleggingssporsmalSvarConsumer(
 ) {
     val consumerProperties = kafkaAivenConsumerConfig<KafkaKartleggingssporsmalSvarDTODeserializer>(
         kafkaEnvironment = kafkaEnvironment,
-        offsetResetStrategy = OffsetResetStrategy.LATEST,
-    )
-    consumerProperties.apply {
+        offsetResetStrategy = OffsetResetStrategy.EARLIEST,
+    ).apply {
+        this[ConsumerConfig.GROUP_ID_CONFIG] = "ismeroppfolging-v0"
```

Comment:
Må bruke en annen consumerId siden vi allerede har fått satt offset for `ismeroppfolging-v1`.